src/DwitTech.AccountService.Core/Services/ActivationService.cs

```diff
@@ -60,12 +56,55 @@ public bool SendActivationEmail(string fromEmail, string toEmail, string templat
             var baseUrl = GetBaseUrl();
             var activationUrl = GetActivationUrl();
             string templateText = GetTemplate(templateName);
-            templateText = templateText.Replace("{{name}}", RecipientName) ;
+            templateText = templateText.Replace("{{name}}", RecipientName);
             templateText = templateText.Replace("{{activationUrl}}", activationUrl);
             string body = templateText;
             var response = SendMail(fromEmail, toEmail, subject, body, cc, bcc);
 
             return response;
         }
+
+        public bool SendWelcomeEmail(string fromEmail, string toEmail, string templateName, string subject = "Account Details", string cc = "", string bcc = "")
+        {
+            var user = new User();
```

Comment:
This is not functional. You should remove this and accept a parameter of type User instead.

Reply:
> I don't see the benefit of this. You should remove it and accept a parameter of type User instead, or something along that line

Resolved